src/main/scala/com/codergaden/ex1/Application.scala

```diff
@@ -0,0 +1,55 @@
+package com.codergaden.ex1
+
+object Application {
+
+  def main(args: Array[String]): Unit = {
+    val res = getShorterLonger(args.toList)
+    println(
+      s"""Shorter word/s: ${res.shortestWords.words.mkString(" ")} with num letters ${res.shortestWords.lengthOfWord}
+      |Longer word/s: ${res.longestWords.words.mkString(" ")} with num letters ${res.longestWords.lengthOfWord}""".stripMargin )
+  }
+
+  def getShorterLonger(wordlist: List[String]): (Result) = {

```

Comment:
I would put getShorterAndLonger inside a separate object (minor) so then you can test it, leaving application only as entry point

Reply:
you can remove ( from Result as return type